broker/setup_broker/lvk/setup_broker.sh

```diff
@@ -51,16 +69,19 @@ if [ "${teardown}" != "True" ]; then
     # create pubsub
     echo "Configuring Pub/Sub resources..."
     gcloud pubsub topics create "${topic_alerts}"
+    gcloud pubsub topics create "${topic_storebigquery}"
 
     # Set IAM policies on resources
     user="allUsers"
     roleid="projects/${GOOGLE_CLOUD_PROJECT}/roles/userPublic"
     gcloud pubsub topics add-iam-policy-binding "${topic_alerts}" --member="${user}" --role="${roleid}"
+    gcloud pubsub topics add-iam-policy-binding "${topic_storebigquery}" --member="${user}" --role="${roleid}"
 
 else
     # ensure that we do not teardown production resources
     if [ "${testid}" != "False" ]; then
         o="GSUtil:parallel_process_count=1" # disable multiprocessing for Macs
+        bq rm -r -f "${PROJECT_ID}:${bq_dataset}"
         gsutil -m -o "${o}" rm -r "gs://${broker_bucket}"
```

Comment:
The `o` variable goes with the `gsutil` command, so don't separate them.

```suggestion
        o="GSUtil:parallel_process_count=1" # disable multiprocessing for Macs
        gsutil -m -o "${o}" rm -r "gs://${broker_bucket}"
        bq rm -r -f "${PROJECT_ID}:${bq_dataset}"
```